src/fstab-generator/fstab-generator.c

```diff
@@ -578,8 +578,39 @@ static int parse_fstab(bool initrd) {
                         if (streq(canonical_where, where))
                                 canonical_where = mfree(canonical_where);
                         else
-                                log_debug("Canonicalized what=%s where=%s to %s",
-                                                what, where, canonical_where);
+                                log_debug("Canonicalized where=%s to %s (for what=%s)",
+                                                where, canonical_where, what);
+                }
+
+                if (fstab_test_option(me->mnt_opts, "bind\0" "rbind\0")) {
```

Comment:
this is not the only way to declare bind mounts: fstype might be bind/rbind too, compare with mount_is_bind() in src/core/mount.c.